tests/integration/test_social_media.py

```diff
@@ -18,12 +18,13 @@
 def test_create_text_post(client):
     user = f.create_user(email='test@example.com', password='test')
     client.login(user=user)
+    sample_time = '2018-10-09T18:30:00Z'
```

Comment:
you can just call it `scheduled_time` or `post_scheduled_time`.

`sample_time` loses the context with respect to test. I have to find what `sample_time` means when I see it, and probably make sense from the change on line 27.
Thoughts?